content/help/ops/security/health-checks-and-mtls/index.md

```diff
@@ -2,7 +2,7 @@
 title: Health Checks and Mutual TLS
 description: How to get health checks working when mutual TLS is enabled.
 weight: 40
+aliases:
+    - /help/ops/setup/app-health-check
```

Comment:
Not quite right.

The alias definition goes into the TARGET file (where you want people to end up) and have the path of the ORIGINAL (the file being deleted).

So you delete the file you want to delete, and then put an alias to that file in the file you want to redirect people to.

Reply:
Done